SECURITY.md

```diff
@@ -0,0 +1,51 @@
+# Reporting security issues
+
+We're extremely grateful to security researchers and users
+who report vulnerabilities to WireMock and its ecosystem.
+All reports are thoroughly investigated by maintainers of respective repositories.
+
+## When to report issues?
+
+- You think you discovered a potential security vulnerability in WireMock or its components
+- You are unsure how a publicly announced vulnerability in an upstream project affects WireMock,
+  and you suspect users might be impacted.
+- You think you discovered a new vulnerability in another project that WireMock depends on.
+  If so, make sure to report the issue to that project too by following their guidelines
+
+Note that security hardening enhancements are not considered vulnerabilities,
+and the common Request for Enhancement issue report should be used.
+
+## How to report an issue?
+
+Most of WireMock repositories use the _Security Advisory Advisory_ portal
```

Comment:
The word Advisory appears twice here, is this correct?

Reply:
```suggestion
Most of WireMock repositories use the _GitHub Security Advisory_ engine
```